### PR TITLE
Generate a lot of things

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @sdroege @EPashkin (sorry, it's really a lot to go through...)

Once this PR merged, I'll move the repository into gtk-rs.

### CODE REVIEW

File: Gir.toml [hunk 1]
```diff
@@ -11,35 +11,68 @@ single_version_file = true
...
     "Atk.Role",
+    "Atk.Selection",
+    "Atk.Socket",
     "Atk.State",
-    "Atk.Table",
+    "Atk.StateType",
+    "Atk.StateSet",
+    "Atk.StreamableContent",
     "Atk.TableCell",
     "Atk.Text",
```
Comment (EPashkin) [inline Gir.toml]: "Atk.Text" this need be removed, as it in separate section

Comment (EPashkin) [reply Gir.toml]: After remove and regen `free_ranges` problem "fixed"

File: Gir.toml [hunk 2]
```diff
@@ -11,35 +11,68 @@ single_version_file = true
...
+    "Atk.Util",
+    "Atk.Value",
+    "Atk.ValueType",
+    "Atk.Window",
 ]
 
 ignore = [
-    "Atk.Relation",
-    "Atk.Plug", # conflicting implementations of trait `glib::IsA<glib::Object>` for type `auto::plug::Plug`
-    "Atk.Window", # conflicting implementations of trait `glib::translate::ToGlibPtr<'_, *mut gobject_ffi::GObject>` for type `auto::window::Window`
```
Comment (EPashkin) [inline Gir.toml]: IMHO "Atk.Plug" and "Atk.Window"'s traits need renaming to remove mixing with gdk's and gtk's

File: src/auto/implementor.rs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use Object;
+use ffi;
+use glib::translate::*;
+use glib_ffi;
+use std::mem;
+use std::ptr;
```
Comment (EPashkin) [inline src/auto/implementor.rs]: Empty file, not regenerated.
4 others too, so need regen after emptying `src\auto`

File: Gir.toml [hunk 4]
```diff
@@ -11,35 +11,68 @@ single_version_file = true
...
+    "Atk.Hyperlink",
+    "Atk.HyperlinkImpl",
+    "Atk.HyperlinkStateFlags",
     "Atk.Hypertext",
     "Atk.Image",
+    # "Atk.Implementor",
+    # "Atk.KeyEventStruct",
     "Atk.Layer",
     "Atk.Misc",
+    # "Atk.NoOpObject",
```
Comment (EPashkin) [inline Gir.toml]: IMHO better uncomment this

File: Gir.toml (hunk 2 above)
Comment (EPashkin) [reply Gir.toml]: Hm, there too many trait duplicates with gtk, now I don't sure that it need renaming.

File: Gir.toml [hunk 5]
```diff
@@ -11,35 +11,68 @@ single_version_file = true
...
+    "Atk.NoOpObjectFactory",
+    "Atk.ObjectFactory",
+    "Atk.Plug",
+    "Atk.PropertyChangeHandler",
+    # "Atk.PropertyValues",
     "Atk.Range",
     "Atk.Rectangle",
```
Comment (EPashkin) [inline Gir.toml]: "Atk.Rectangle" - Boxed type without methods

File: Gir.toml (hunk 1 above)
Comment (GuillaumeGomez) [reply Gir.toml]: Forgot to regen before pushing, my bad...

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
There still empty src/auto/implementor.rs and src/auto/property_values.rs.
Other that that all looks good.

**Comment by GuillaumeGomez** [comment]
Removed the empty files for good this time.

**Pull request merged**